src/utils/dom.js

```diff
@@ -102,16 +103,16 @@ export function removeChildren(parent) {
 
 /**
  * Injects the provided styles in the HEAD section of the document.
- * @param {!Document} doc The document object.
+ * @param {!Doc} doc The document object.
  * @param {string} styleText The style string.
  * @return {!Element}
  */
 export function injectStyleSheet(doc, styleText) {
-  const styleElement = createElement(doc, 'style', {
+  const styleElement = createElement(doc.getRootNode(), 'style', {
```

Comment:
This should be `createElement(doc.getWin().document)`. Unfortunately, `ShadowRoot` interface does not provide `createElement` API. A bit surprised the compiler is not complaining about this :(